multi/gr716-multi/gr716-multi.h

```diff
@@ -21,7 +21,8 @@
 
 /* clang-format off */
 /* IDs of special files OIDs */
-enum { id_gpio0 = 0, id_gpio1, id_spi0, id_spi1, id_uart0, id_uart1, id_uart2, id_uart3, id_uart4,
+enum { id_pseudoNull = 0, id_pseudoZero, id_pseudoFull, id_pseudoRandom,
```

Comment:
It's a good idea to add new stuff at the end, to preserve ABI compatibility.

I hope that gpio doesn't assume somewhere `id_gpio0 = 0` (I don't think it does though)

Reply:
Yup, I see that in case of (gr716-multi) devices are created using string LUT `multi_devs[]`, and the last name is `adc0` but id's are `id_adc0-7`. So, I wanted that name match its id (I was unsure `adc0` some gap, and then pseudo devices - also I can't test it on target). In case of imxrt-multi I was able to easly add stuff at the end, as you write.

Reply:
Yeah, but the `multi_devs[]` LUT is only used for creating dev files, it's not indexed by `id_` in any place - and e.g. not all ADCs are here because of pin confilcts if we were to enable them. So I think it's safe to just add the strings in the LUT directly after `"adc0"`

Reply:
Also there's already no mapping in `id` :arrow_right: `multi_devs[]` - with `uart` and `console`

Reply:
but (just wondering) from the other hand these pseudo devices are available always ("common" posix devices), while gpios, adcs, etc may be setup/sorted differently. So, it is legit to put them (common) first, so later (common) ABI does not change.

Reply:
We don't want pseudo devices to be available always - to be able to use posixsrv if we decide to

Reply:
ack